src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBiddingClosedMessage.tsx

```diff
@@ -43,13 +33,8 @@ const BiddingClosedMessage: React.FC<BiddingClosedMessageProps> = ({
           {newAlertModalEnabled ? (
             <>
               <ProgressiveOnboardingAlertCreateSimple>
-                <CreateAlertButton
-                  width="100%"
-                  size="large"
-                  onClick={showAlert}
-                />
+                <CreateAlertButton width="100%" size="large" />
               </ProgressiveOnboardingAlertCreateSimple>
-              {alertComponent}
```

Comment:
**question:** Are you removing `alertComponent` here because it's already present in the Sidebar?

Reply:
Instead of explicitly rendering this modal/alert component next to each `Create Alert` button the idea is to place an AlertProvider component at a point in the component tree which 1) has access to the desired initialCriteria 2) renders one instance of the modal component alongside its children 3) will wrap any other branches of the component tree which might need to dispatch the modal via (Create Alert) buttons.

So this PR hoists the modal component up not only from the sidebar but from any deeper spot in the component tree.

Reply:
Great, thanks for the explanation 🙏